pixleesdk/src/main/java/com/pixlee/pixleesdk/client/PXLClient.kt

```diff
@@ -22,6 +22,10 @@ class PXLClient(val context: Context) {
         var secretKey: String? = null
         var android_id: String? = null
 
+        var apiTimeoutRead: Long? = null
+        var apiTimeoutConnect: Long? = null
+        var apiTimeoutWrite: Long? = null
```

Comment:
Prefer default numbers here instead of null
```suggestion
        var apiTimeoutRead: Long = 60
        var apiTimeoutConnect: Long = 60
        var apiTimeoutWrite: Long = 180
```